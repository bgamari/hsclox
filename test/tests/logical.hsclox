print !false;
print !true;
print !nil;
