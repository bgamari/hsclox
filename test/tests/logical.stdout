== <script> ==
0000    1 OP_FALSE
0001    | OP_NOT
0002    | OP_PRINT
0003    2 OP_TRUE
0004    | OP_NOT
0005    | OP_PRINT
0006    3 OP_NIL
0007    | OP_NOT
0008    | OP_PRINT
0009    4 OP_NIL
0010    | OP_RETURN
true
false
true
